### ISSUE TITLE
Support for Personal Repos

### ISSUE THREAD

**Issue by ashfurrow** [CircleCI-Public/find-circle-yml#1]
I know this tool is meant to look for all repos in an organization that are still on Circle CI 1.0, but I have a lot of personal repos and it'd be nice to use it for those, too.

**Comment by transdigiware** [comment]
apa beda CircleCI 2.0  tentang Lingkar CI 1.0, 
dalam pemahaman saya cuma wadah ikut serta migrasi saja tanpa kita pernah tahu bagaimana  itu Mendapatkan green build itu, karena untuk bercengkrama dengan CI 1.0 saja belum sempurna terlebih migrate now to CI 2.0. ? 
How about everyone !

**Comment by fatso83** [comment]
Yeah, I don't understand how to make this just work on personal repos. I would like to submit a PR, but I see that you have had PRs lingering without any response for a year (#4), so not wasting my time.
